bitnami/kube-prometheus/templates/exporters/kube-controller-manager/endpoints.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if and .Values.kubeControllerManager.enabled .Values.kubeControllerManager.endpoints }}
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: {{ template "kube-prometheus.fullname" . }}-kube-controller-manager
+  namespace: kube-system
```

Comment:
it should be `{{ .Values.kubeControllerManager.namespace }}`

Reply:
corrected